StereoKitC/platforms/linux.cpp

```diff
@@ -426,10 +421,14 @@ bool linux_start_pre_xr() {
 ///////////////////////////////////////////
 
 bool linux_start_post_xr() {
-	#if defined(SKG_LINUX_EGL)
-	if (!sk_settings.disable_desktop_input_window && !setup_x_window())
+#if defined(SKG_LINUX_EGL)
+	if (sk_settings.disable_desktop_input_window)
+		return true;
+	if(skg_egl_dri())
```

Comment:
I just don't love these calls to `skg_egl_dri`. Somewhat related to the comment below, maybe `setup_x_window` can have a mode where failure is okay, and doesn't log anything?